static/js/components/input/ProductSelector.js

```diff
@@ -167,26 +167,25 @@ export class ProductSelector extends React.Component<Props, State> {
     if (productType === PRODUCT_TYPE_PROGRAM) {
       return buildProgramDateOptions(
         programRuns.filter(programRun => {
-          let startDate = null
-          if (programRun.start_date) {
-            startDate = new Date(programRun.start_date)
+          let endDate = null
+          if (programRun.end_date) {
+            endDate = new Date(programRun.end_date)
           }
-          return startDate && startDate >= todaysDate
+          return endDate && endDate >= todaysDate
```

Comment:
Is this supposed to be `(endDate === null || endDate >= todaysDate)` similar to the expression below?

Reply:
@noisecapella  Yes, that makes sense, We usually treat end dates to be in the future if none mentioned. Right?

For this there would be a little change in ecommerce/views.py (e.g.
`valid_runs=Count("programruns", filter=Q(programruns__end_date__gt=now))`
-> `valid_runs=Count("programruns", filter=Q(programruns__end_date__gt=now) | Q(programruns__end_date=None))`).
Should I go ahead with this change as well?

Reply:
@noisecapella I did some A/B testing and to confirm it as well. Finally did these requested changes and updated the associated test/API as well.